tensorrt_llm/_torch/modules/attention.py

```diff
@@ -119,13 +125,14 @@ def __init__(
             self.hidden_size,
             bias=self.dense_bias,
             dtype=dtype,
-            parallel_config=ParallelConfig(
-                tensor_parallel_size=tp_size,
-                tensor_parallel_rank=tp_rank,
-                tensor_parallel_mode=TensorParallelMode.ROW,
+            mapping=Mapping(
```

Comment:
Use the same Mapping object created on line 110

Reply:
Done